labelbox/data/serialization/labelbox_v1/objects.py

```diff
@@ -42,6 +46,52 @@ def validate_subclasses(cls, value, field):
         return value
 
 
+class _Coordinates(BaseModel):
+    """Union of the possible coordinate lists in tiled imagery exports"""
+    coordinates: Union[List[float], List[List[float]], List[List[List[float]]]]
+
+
+class LBV1TIPoint(LBV1ObjectBase):
+    object_type: Literal['point'] = Field(..., alias='type')
+    geometry: _Coordinates
+
+    def to_common(self) -> Point:
+        lng, lat = self.geometry.coordinates
+        return Point(x=lng, y=lat)
+
+
+class LBV1TILine(LBV1ObjectBase):
+    object_type: Literal['polyline'] = Field(..., alias='type')
+    geometry: _Coordinates
+
+    def to_common(self) -> Line:
+        return Line(points=[
+            Point(x=coord[0], y=coord[1]) for coord in self.geometry.coordinates
+        ])
+
+
+class LBV1TIPolygon(LBV1ObjectBase):
+    object_type: Literal['polygon'] = Field(..., alias='type')
+    geometry: _Coordinates
+
+    def to_common(self) -> Polygon:
+        for coord_list in self.geometry.coordinates:
+            return Polygon(
+                points=[Point(x=coord[0], y=coord[1]) for coord in coord_list])
+
+
+class LBV1TIRectangle(LBV1ObjectBase):
+    object_type: Literal['rectangle'] = Field(..., alias='type')
+    geometry: _Coordinates
+
+    def to_common(self) -> Rectangle:
+        coord_list = self.geometry.coordinates[0]
+        start = coord_list[0]
+        end = coord_list[2]
```

Comment:
These positions aren't guaranteed since the bounding box is represented as a polygon. Maybe we should find the min/max x/y values to be used. That way this can't silently break